build/accelerate_launch.py

```diff
@@ -47,6 +48,40 @@
 ERROR_LOG = "/dev/termination-log"
 
 
+def copytree(source, destination, symlinks=False, ignore=None):
+    if not os.path.exists(destination):
+        os.makedirs(destination)
+        shutil.copystat(source, destination)
+    file_list = os.listdir(source)
+    # If we want to ignore any files
+    if ignore:
+        exclude = ignore(source, file_list)
```

Comment:
I like the idea of allowing files to ignore although this is not a feature we are yet adding as this would require parsing the JSON the user passes in and adding a new flag for files to ignore. In addition, what is this `ignore()` method?